modin/pandas/accessor.py

```diff
@@ -191,3 +195,71 @@ def __get__(self, obj, cls):
         accessor_obj = self._accessor(obj)
         object.__setattr__(obj, self._name, accessor_obj)
         return accessor_obj
+
+
+class ExperimentalFunctions:
+    """
+    Namespace class for accessing experimental Modin functions.
+
+    Parameters
+    ----------
+    data : DataFrame or Series
+        Object to operate on.
+    """
+
+    def __init__(self, data):
+        warnings.warn(
+            "Thank you for using the Modin Experimental pandas API."
```

Comment:
Do we really need to emit this warning?

Reply:
Removed here as well as for importing `modin.experimental.pandas`. 
This is a debatable question, but I want to make it consistent. I also believe that the name of the module speaks for itself.